README.md

```diff
@@ -20,6 +19,18 @@ This includes the following core Silverstripe CMS modules:
 This can be either added to an existing project or used as a project base for creating a
 basic Silverstripe CMS project.
 
+*Unlike other Silverstripe recipes, this recipe does not constrain Silverstripe core dependencies to a particular minor version line.*
+
+If you wish to use a specific minor version of Silverstripe, you can specify a version of `silverstripe/framework` in your project's `composer.json` file with tilde `~`. For example:
```

Comment:
```suggestion
If you wish to use a specific minor version of Silverstripe CMS, you can specify a version of `silverstripe/framework` in your project's `composer.json` file with tilde `~`. For example:
```

This would only restrict framework, wouldn't it? I think [inlining the recipe](https://github.com/silverstripe/recipe-plugin?tab=readme-ov-file#inlining-recipes) should be the recommendation instead.

Reply:
It'll restrict framework, which will then restrict the version of recipe-core (requirement of recipe-ccl) that's installable which is lockstepped and it'll cascade from there.

I don't think inlining is correct in this situation because then you'll just end up with the contents of recipe-ccl's composer.json require section in your projects composer.json.  At that point you'd then need to manually restrict-core which is no different from my recommendation, expect it has an extra step and you're dealing with recipe-core instead of framework which I'm assuming far less people are familiar with.

Reply:
That doesn't cover these dependencies though:

>         "cwp/cwp-core"
>         "silverstripe/auditor"
>         "silverstripe/environmentcheck"
>         "silverstripe/hybridsessions"

Reply:
^ still needs resolving.

You either need to add all of the dependencies manually so that they're all managed, or just inline the recipe which does that for you.

Reply:
Alternatively don't give advice about this at all, since the AC is just to say this recipe is different from the others, not how to manage your dependency constraints.

Reply:
> Alternatively don't give advice about this at all, since the AC is just to say this recipe is different from the others, not how to manage your dependency constraints.

We'll just go with this